x-pack/plugins/triggers_actions_ui/public/application/alerts_app.tsx

```diff
@@ -29,23 +27,18 @@ export const renderApp = (deps: TriggersAndActionsUiServices) => {
 };
 
 export const App = ({ deps }: { deps: TriggersAndActionsUiServices }) => {
-  const { dataViews, theme, theme$ } = deps;
-  const isDarkMode = theme.getTheme().darkMode;
```

Comment:
This is just passing through the dark mode setting from Core, which isn't explicitly needed with `KibanaRenderContextProvider`